### PR TITLE
fix test main setup in client/remote_test.go

### PR DESCRIPTION
**Pull request by lziest**

### CODE REVIEW

File: client/remote_test.go [hunk 1]
```diff
@@ -39,85 +40,59 @@ var (
...
+		s.UnixListenAndServe()
 	}()
 
 	// wait for server to start
 	time.Sleep(100 * time.Millisecond)
 	// Setup keyless client
-	if c, err = NewClientFromFile(clientCert, clientKey, keyserverCA); err != nil {
-		t.Fatal(err)
-	}
+	c, _ = NewClientFromFile(clientCert, clientKey, keyserverCA)
```
Comment (kisom) [inline client/remote_test.go]: Can you add a comment explaining why errors aren't being checked?

Comment (lziest) [inline client/remote_test.go]: I haven't figured out how to properly bail out setup errors in TestMain. Or maybe I am using it wrong.

**Comment by lziest** [comment]
realized that I can do log.Fatal(), so all errors are reported.

**Comment by kisom** [comment]
LGTM.

**Pull request merged**